README.rst

```diff
@@ -211,12 +211,39 @@ It is possible to do batch requests using a Python context manager (``with``):
 .. code-block:: python
 
   with client.batch() as batch:
-      for idx in range(0,100): 
+      for idx in range(0,100):
           batch.update_record(data={'id': idx})
 
 A batch object shares the same methods as another client.
 
 
+Retry on error
+--------------
+
+When the server is throttled, under heavy load, or maintenance, it can
```

Comment:
consider `When the server is throttled (under heavy load or maintenance), it can return error responses.`